nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -42,6 +42,14 @@ def test_find_cut_coords():
         np.array([x, y, z]),
         0.5 * np.array(data.shape).astype(np.float))
 
+    # regression test (cf. #922)
+    # pseudo-4D images as input (i.e., X, Y, Z, 1)
+    # previously raised "ValueError: too many values to unpack"
+    data = np.ones((36, 43, 36))[..., np.newaxis]
+    affine = np.eye(4)
+    img = nibabel.Nifti1Image(data, affine)
+    x, y, z = find_xyz_cut_coords(img, activation_threshold=1.1)
+
```

Comment:
You don't test the output ?
x, y, z is supposed to be what ? To me the result is empty for activation_threshold=1.1